pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -680,6 +682,14 @@ func cleanUpController(ctx context.Context, newSCSIController types.BaseVirtualD
 	return nil
 }
 
+// Get datastore from VMDisk path.
+// vmDiskPath format is [<datastore> kubevols/<volume>]
+func getDatastoreFromVMDiskPath(vmDiskPath string) string {
```

Comment:
I don't want to replicate the string parsing logic given that govmomi has helpers for it now.